apps/browser/qml/pages/PermissionExceptionsPage.qml

```diff
@@ -92,7 +96,24 @@ Page {
                 MenuItem {
                     //% "Delete"
                     text: qsTrId("sailfish_browser-me-delete")
-                    onClicked: remove(model.index)
+                    onClicked: remove(model.uri, model.type, model.capability)
+                }
+            }
+        }
+
+        PullDownMenu {
+            visible: model && (view.count > 0)
+            MenuItem {
+                //% "Delete all exceptions"
+                text: qsTrId("sailfish_browser-me-delete-all-exceptions")
+                onClicked: {
+                    remorse = Remorse.popupAction(
+                                page,
+                                //% "Deleted exceptions"
+                                qsTrId("sailfish_browser-deleted-exceptions"),
+                                function() {
+                                    page.model.removeAllForPermissionType(proxyModel.permissionType)
+                                })
                 }
             }
         }
```

Comment:
So menu order should be..

"Delete all exceptions"
"Add new"

As I'd expect "add new" be more used... and this order it's easier to reach.

Reply:
This still